db/models.py

```diff
@@ -50,3 +57,62 @@
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey("User", on_delete=models.CASCADE)
```

Comment:
The `user` field in the `Order` model references a `User` model that is defined later in the file. Ensure that the `User` model is correctly referenced, possibly by using `settings.AUTH_USER_MODEL` if you are using a custom user model.